egs2/chime8_task1/asr1/README.md

```diff
@@ -0,0 +1,473 @@
+
+**This recipe contains only the baseline for the acoustic robustness sub-track (ASR-only, oracle diarization).** <br>
+The recipe for the main track is in `egs2/chime7_dasr/diar_asr1/README.md`.
+
+Since here we leverage oracle diarization the diarizer component here is omitted.
+The baseline system for this sub-track is composed of two main modules (with an optional channel selection module):
+1. Guided Source Separation (GSS) [5], here we employ the GPU-based version (much faster) from [Desh Raj](https://github.com/desh2608/gss).
+2. End-to-end ASR model based on [4], which is a transformer encoder/decoder model trained <br>
+with joint CTC/attention [6]. It uses WavLM [7] as a feature extractor.
+3. Optional Automatic Channel selection based on Envelope Variance Measure (EV) [8].
+
+### 3.1 Results
+
+#### 3.1.1 Main Track
+See `egs2/chime7_dasr/diar_asr1/README.md`.
+The result of the diarization system is fed to this recipe GSS+ASR pipeline.
+
+#### 3.1.2 Acoustic Robustness Sub-Track: Oracle Diarization + ASR
+Pretrained model: [popcornell/chime7_task1_asr1_baseline](popcornell/chime7_task1_asr1_baseline) <br>
+Detailed decoding results (insertions, deletions etc) are available in `baseline_logs/RESULTS.md` here. <br>
+
+Note that WER figures in `baseline_logs/RESULTS.md` and in the model card in [popcornell/chime7_task1_asr1_baseline](popcornell/chime7_task1_asr1_baseline) will
+be different slighly (a bit higher) from the diarization-attributed WER (DA-WER) score we use for ranking. This latter, as cpWER [1], is based on concatenated utterances for each speaker.
+The final score is described in Section 4 and is similar to the one used in previous CHiME-6 Challenge except
+that it is not permutation invariant now, but the speaker mapping is instead assigned via diarization (hence diarization-assigned WER).
+
+Here we report the results obtained using channel selection (retaining 80% of all channels) prior to performing GSS and decoding with the baseline pre-trained
+ASR model. This is the configuration that gave the best results overall on the dev set.
+
+<table>
+<thead>
+  <tr>
+    <th>Dataset</th>
+    <th>split</th>
+    <th>front-end</th>
+    <th>DA-WER (%)</th>
+    <th>macro DA-WER (%)</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>CHiME-6</td>
+    <td rowspan="3">dev</td>
+    <td>GSS (EV top 80%)</td>
+    <td> 32.64 </td>
+    <td rowspan="3"> 28.81 </td>
+  </tr>
+  <tr>
+    <td>DiPCo</td>
+    <td>GSS (EV top 80%)</td>
+    <td> 33.54 </td>
+  </tr>
+  <tr>
+    <td>Mixer-6</td>
+    <td>GSS (EV top 80%)</td>
+    <td> 20.25 </td>
+  </tr>
+</tbody>
+</table>
+
+
+Such baseline system would rank third on dev set based on the rules of the past CHiME-6 Challenge
+on Track 1 (unconstrained LM).
+Results on the evaluation set will be released after the end of the CHiME-7 DASR Challenge. <br>
```

Comment:
Should this description be updated?

Reply:
Yes thanks ! i removed this file actually